user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+
+- Tools to collect and process requests received via email or from a website form
+
+Please see the :ref:`Sales Tools section <user-guide-sales-index>` of the User Guide for more details.
+
+  
+Tools for Marketing Associates 
+------------------------------
+
+Marketing associates are responsible for growing a business's customer base.  This is sometimes done through email 
+marketing, customer segmentation, and campaign analysis.  OroCRM facilitates all of these actions and more by providing:
+
+- Segmentation on a specific set of customers by nearly any field  in OroCRM
+
+- Ability to create dynamic targeted marketing lists specific to customer segments
+
+- Ability to send personalized emails using customer data with customizable templates from within OroCRM 
+
+- Automatic synchronization of dynamic marketing lists with 3rd party email marketing platforms such as Mailchimp or 
+  Dotmailer to ensure your contact list is always up to date
+  
+- Tracking of online and offline marketing campaign results in order to study their outcome and effectiveness
+
+See the :ref:`Marketing Tools section <user-guide-marketing>` of the User Guide to learn more about how OroCRM can help 
+with marketing-related activities.
+
+
+
+Tools for Helpdesk and Support 
+------------------------------
+
+Customer service is critical for customer retention in the retail industry.  In fact most consumers say they view 
+customer service as the true test of how much a company values them.  OroCRM can help you maximize customer satisfaction 
+by:
+
+
+- Integrating with Zendesk or other ticket management systems 
+
+- Keeping track of tickets raised by customers so all teams are aware of an account's status
+
+- Contacting the customer via direct email, voice calls, or a Google Hangout
+
+- Allowing the creation of unique workflows and processes that match your business’s support process
+
+See the :ref:`Support Tools section <user-guide-support-index>` of this guide to learn more about how OroCRM can help 
+with support-related activities.
+
+
+Tools for the System Configurator and Administrator
+---------------------------------------------------
+
+OroCRM is infinitely configurable and provides tools that allow system configurators to tailor the system in accordance 
+with the business’s needs. OroCRM also provides tools for day to day administration of the system. These tools include:
+
+- Personalizing and localizing the system
+
+- Setting up the organization structure
+
+- Creation and management of user profiles
+
+- Configuration of permissions and access rights with Roles
+
+- Creation and configuration of new and existing entities
+
+- Configuration of workflows
+
+- Ability to set up and manage integrations and extensions (with support of the Oro team for SaaS environments)
+
+  
+See the system :ref:`System Management and Administration section <user-guide-admin-tools>` to learn more about what 
+configurators and administrators can achieve with OroCRM.
```

Comment:
Edit: See the system :ref:`System Management and Administration section <user-guide-admin-tools>` to learn more about what else administrators can achieve with OroCRM.